src/TimeService/timeservice-rtems.h

```diff
@@ -0,0 +1,49 @@
+#include "interfaces_info.h"
+
+#define TICKS_PER_SECOND 1000000
+#define SCALER_VALUE 5
+
+void init_timer(Timer_Apbctrl1 *const timer, bool isChained, Timer_Id timerId)
```

Comment:
Do we need to restart timers at all? We have two timers running in chained configuration right now in kazoo, they start always when starting the program. Restarting them may lead to problems in my opinion. Moreover i think it is unnecessary if as far as i know in TimeService we only measure the passage of time.

Reply:
In the middle of the development I noticed that the clock is returning 0 and restarting timers helped but now I see that it somehow works without that, so I will remove it.